### PR TITLE
12.0 mig web_widget_color

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Tested on runbot Thanks! :+1:

File: web_widget_color/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Copyright (C) 2014 Anybox <http://anybox.fr>
+# Copyright (C) 2015 Taktik SA <http://taktik.be>
+# Copyright (C) 2018 Alexandre Díaz <dev@redneboa.es>
+#
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).#
+{
+    'name': "Web Widget Color",
+    'category': "web",
+    'version': "12.0.1.0.0",
+    "author": "Savoir-faire Linux, "
```
Comment (nikul-serpentcs) [inline web_widget_color/__manifest__.py:12]: IMO add `website` key, and URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: web_widget_color/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+              "Anybox, "
+              "Taktik SA, "
+              "Sudokeys, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    'depends': ['base', 'web'],
```
Comment (nikul-serpentcs) [inline web_widget_color/__manifest__.py]: In `web`, `base` already depends, So remove `base`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
All comments have been honored, so I merge.

**Pull request merged**